src/pages/signin.tsx

```diff
@@ -175,14 +190,24 @@ const SignIn: NextPage = () => {
             {/*   <Button.Icon icon={FcGoogle} /> */}
             {/*   Entrar com Google */}
             {/* </Button> */}
-            <Button
-              onClick={() => handleStrategyLogin("/auth/linkedin")}
-              size="small"
-              variant="terciary"
-            >
-              <Button.Icon icon={GrLinkedinOption} className="text-[#0e76a8]" />
-              Entrar com Linkedin
-            </Button>
+            <div className="flex flex-col gap-4">
+              {LOGIN_OPTIONS.map((option) => {
+                return (
+                  <Button
+                    size="small"
+                    variant="terciary"
+                    onClick={() => handleStrategyLogin(option.authPath)}
+                    key={Math.random()}
+                  >
```

Comment:
aqui seria melhor pegar o index do map para gerar a key to botão, isso gera uma falha de segurança
![image](https://github.com/Mentor-Cycle/mentor-cycle-fe/assets/57500163/20696512-8ae4-4e42-8842-cdd19d2c304e)